### ISSUE TITLE
Add names to select elements

### ISSUE THREAD

**Issue by brentryanjohnson** [ONRR/doi-extractives-data#3309]
Our accessibility audit flagged missing names for select elements.

## `all-production`
```
 • Error: This select element does not have a name available to an accessibility API. Valid names are: label element, ti
tle attribute, aria-label attribute, aria-labelledby attribute.
   ├── WCAG2AA.Principle4.Guideline4_1.4_1_2.H91.Select.Name
   ├── #all-production > h3 > select
   └── <select class="chart-selector flex-row-icon"> ...</select>
```

## `federal-production`
```
 • Error: This select element does not have a name available to an accessibility API. Valid names are: label element, ti
tle attribute, aria-label attribute, aria-labelledby attribute.
   ├── WCAG2AA.Principle4.Guideline4_1.4_1_2.H91.Select.Name
   ├── #federal-production > section > h3 > select
   └── <select class="chart-selector flex-row-icon"> ...</select>
```

## `revenue-trends`
```
• Error: This select element does not have a name available to an accessibility API. Valid names are: label element, ti
tle attribute, aria-label attribute, aria-labelledby attribute.
   ├── WCAG2AA.Principle4.Guideline4_1.4_1_2.H91.Select.Name
   ├── #revenue-trends > select
   └── <select class="chart-selector flex-row-icon"> ...</select>
```

## `gdp`
```
 • Error: This select element does not have a name available to an accessibility API. Valid names are: label element, ti
tle attribute, aria-label attribute, aria-labelledby attribute.
   ├── WCAG2AA.Principle4.Guideline4_1.4_1_2.H91.Select.Name
   ├── #gdp > h3 > select
   └── <select class="chart-selector flex-row-icon"> ...</select>
```

## `employment`
```
 • Error: This select element does not have a name available to an accessibility API. Valid names are: label element, ti
tle attribute, aria-label attribute, aria-labelledby attribute.
   ├── WCAG2AA.Principle4.Guideline4_1.4_1_2.H91.Select.Name
   ├── #employment > section:nth-child(1) > h3 > select
   └── <select class="chart-selector flex-row-icon"> ...</select>
```

## `self-employment`
```
 • Error: This select element does not have a name available to an accessibility API. Valid names are: label element, ti
tle attribute, aria-label attribute, aria-labelledby attribute.
   ├── WCAG2AA.Principle4.Guideline4_1.4_1_2.H91.Select.Name
   ├── #self-employment > select
   └── <select class="chart-selector flex-row-icon"> ...</select>
```

**Comment by mcharg** [comment]
Closing in favor of new accessibility audit.
